src/main/java/org/jabref/gui/importer/ImportEntriesViewModel.java

```diff
@@ -119,10 +119,10 @@ public void importEntries(List<BibEntry> entriesToImport, boolean shouldDownload
                                                      .anyMatch(this::hasDuplicate)).onSuccess(duplicateFound -> {
                 if (duplicateFound) {
                     boolean continueImport = dialogService.showConfirmationDialogWithOptOutAndWait(Localization.lang("Duplicates found"),
-                            Localization.lang("There are possible duplicates (marked with an icon) that haven't been resolved. Continue?"),
-                            Localization.lang("Continue with import"),
+                            Localization.lang("There are possible duplicates that haven't been resolved. Continue?"),
+                            Localization.lang("Auto merge"),
                             Localization.lang("Cancel import"),
-                            Localization.lang("Disable this confirmation dialog"),
+                            Localization.lang("Do not ask again"),
```

Comment:
Please replace in all files where this string is used:

Hint: Use <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>F</kbd> to search in all files

![grafik](https://user-images.githubusercontent.com/1366654/103555192-2b252f80-4eb0-11eb-9b21-c3e66716d5cd.png)

Reply:
Done